aws-xray-recorder-sdk-aws-sdk/src/main/resources/com/amazonaws/xray/handlers/DefaultOperationParameterWhitelist.json

```diff
@@ -325,6 +325,409 @@
           ]
         }
       }
+    },
+    "Amazon S3": {
+      "operations": {
+        "CopyObject": {
+          "request_parameters": [
+            "SourceBucketName",
+            "SourceKey",
+            "DestinationBucketName",
+            "DestinationKey"
+          ]
+        },
+        "CopyPart": {
+          "request_parameters": [
+            "SourceBucketName",
+            "SourceKey",
+            "DestinationBucketName",
+            "DestinationKey"
+          ]
+        },
+        "GetObject": {
+          "request_parameters": [
+            "Key",
+            "VersionId",
+            "BucketName"
+          ]
+        },
+        "PutObject": {
+          "request_parameters": [
+            "Key",
+            "BucketName"
+          ]
+        },
+        "GetObjectAcl": {
+          "request_parameters": [
+            "Key",
+            "VersionId",
+            "BucketName"
+          ]
+        },
+        "CreateBucket": {
+          "request_parameters": [
+            "BucketName"
+          ]
+        },
+        "ListObjectsV2": {
```

Comment:
Would be interesting to add `KeyCount` as a `response_parameter`:

```json
          "response_parameters": [
            "KeyCount"
          ]
```

Likely won't work on the regular `ListObjects` API (non v2) because of the legacy `ObjectListing` return type.